jujugui/static/gui/src/app/views/utils.js

```diff
@@ -412,13 +412,38 @@ YUI.add('juju-view-utils', function(Y) {
     return regex.test(relationId);
   };
 
-  utils.getRelationDataForService = function(db, service) {
+  /**
+     Return relation objects attached to the application
+
+     @method getRelationDataForService
+     @static
+     @param {Database} db to resolve charms/services on.
+     @param {Object} application in which we want the relations attached too
+     @param {Object} withApplication an optional parameter used to filter the
+      returning array by relations that match both endpoints
+   */
+  utils.getRelationDataForService = function(db, application, withApplication) {
```

Comment:
It might be nice to try and not add more code using the old names (service/application etc.) soon. Not something for this branch, but this code made me think about that as a requirement for the future.